airbyte-scheduler/app/src/main/java/io/airbyte/scheduler/app/JobSubmitter.java

```diff
@@ -133,6 +140,19 @@ void submitJob(final Job job) {
 
           if (output.getStatus() == io.airbyte.workers.JobStatus.SUCCEEDED) {
             persistence.succeedAttempt(job.getId(), attemptNumber);
+
+            final List<StandardWorkspace> standardWorkspaces = configRepository.listStandardWorkspaces(false);
+
+            final List<StandardWorkspace> workspacesToUpdate = standardWorkspaces.stream()
+                .filter(standardWorkspace -> !Optional.ofNullable(standardWorkspace.getFirstCompletedSync()).orElse(false))
+                .map(standardWorkspace -> {
+                  standardWorkspace.setFirstCompletedSync(true);
+                  return standardWorkspace;
+                })
+                .collect(Collectors.toList());
+
+            configRepository.writeStandardWorkspaces(workspacesToUpdate);
+
             if (job.getConfigType() == ConfigType.SYNC) {
```

Comment:
The above should only go in the `SYNC` here. Otherwise check connection will trigger this.

Reply:
Done